frontend/src/components/App/App.css

```diff
@@ -15,7 +15,7 @@
 */
 
 body {
-  font: 400 13px/21px Roboto, -apple-system, BlinkMacSystemFont, "Segoe UI", "Helvetica Neue", Helvetica, sans-serif;
```

Comment:
This is changing everywhere and not just the about. Can I just steal you satellite link?